README.md

```diff
@@ -110,7 +110,7 @@ LizardByte has the full documentation hosted on [Read the Docs](https://docs.liz
         </td>
     </tr>
     <tr>
-        <td>Nvidia: GeForce GTX 1080 or higher</td>
+        <td>Nvidia: GeForce RTX 2000 series or higher</td>
```

Comment:
Changing CUDA version affects Linux only, for Windows the previous GPU is still fine so we probably need to something like the Intel section just above.